operational/aws/instance_scheduled_events/aws_instance_scheduled_events.pt

```diff
@@ -2,18 +2,15 @@ name "AWS Instance Scheduled Events"
 rs_pt_ver 20180301
 type "policy"
 short_description "Report on any AWS scheduled event that will impact instance availability. \n See the [README](https://github.com/rightscale/policy_templates/tree/master/operational/aws/instance_scheduled_events) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
-long_description "Version: 1.0"
+long_description ""
 category "Operational"
 severity "medium"
-
-###############################################################################
-# Permissions
-###############################################################################
-
-permission "perm_read_creds" do
-  actions   "rs_cm.show_sensitive","rs_cm.index_sensitive"
-  resources "rs_cm.credentials"
-end
+info(
```

Comment:
I have another PR that takes care of the cred and info changes.  I dont know if there will be a conflict